pants_plugins/sendwave/pants_docker/python_requirement.py

```diff
@@ -93,12 +90,56 @@ async def get_requirements(
     constraint_arg = ""
     if setup.requirement_constraints:
         constraint_arg = f"--constraint {setup.requirement_constraints}"
+    return PipInstallArgs(
+        links_args=links_args,
+        index_args=index_args,
+        constraint_arg=constraint_arg,
+    )
+
+
+@dataclass(frozen=True)
+class PythonRequirementsFS:
+    requirements: Tuple[PythonRequirementsField]
+
+
+@rule
+async def get_requirements(
+    field_set: PythonRequirementsFS, setup: PythonSetup, repos: PythonRepos
+) -> DockerComponent:
+    install_args = _get_install_args(setup, repos)
+    commands = (
+        f"RUN python -m pip install {install_args.index_args} "
+        + f"{install_args.links_args} {install_args.constraint_arg} "
+        + " ".join(str(lib) for lib in field_set.requirements)
+        + "\n"
+    )
+    return DockerComponent(
+        commands=commands,
+        sources=None,
+    )
 
+
+@dataclass(frozen=True)
+class MultilinePythonRequirementsFS:
```

Comment:
same comment here re:renaming!

Reply:
Done in 9f7d0fa5e1dc28309e6305be4b68c3d0cc1e529 👍🏻